### DISCUSSION TITLE
Scale Image Instead of Cropping?

### DISCUSSION THREAD

**Discussion by robertkstarr** [firebase/extensions#1883] [Q&A] [answered]
It looks like the extension is cropping images instead of scaling them like it's supposed to. Is there an option I need to set to change this?

![1000008164](https://github.com/firebase/extensions/assets/39703492/0e1c14c1-37a7-46da-9df1-6d11a8561408)
![image](https://github.com/firebase/extensions/assets/39703492/04cdcf96-78fa-4a9f-86d7-9f55b582fb13)

**Comment by cleangone** [comment]
I am having the same problem.  This was not an issue with an older version (0.1.12), which scaled the image down but preserved aspect ratio.  The docs indicate there is a max w/h configuration, but I don't see it

**Reply by cleangone** [reply]
I created an issue for this, as the OP has not gotten a response in 2 weeks - https://github.com/firebase/extensions/issues/1891

**Comment by cleangone** [comment, marked as answer]
Mais from the firebase team got back to us with the fix.  Specify this param in the extension config
![resize-config](https://github.com/firebase/extensions/assets/22899277/76f87794-d3bd-414b-84a6-275493d237ef)
